pytorch_toolkit/formula_recognition/datasets/sample_medium_dataset/formulas.norm.lst

```diff
@@ -0,0 +1,134 @@
+\cot ( ( 6.5 W ^ { - 9 } n ^ { 9 } U ^ { 8 } \times 5996 b + 99 d ^ { 8 } F ^ { 79 } ) \times \sqrt { ( 7383 ) } )
```

Comment:
datasets are stored in `training_extensions/data`

Reply:
Moved dataset to this directory